packages/react-jsx-highcharts/src/components/Series/Series.js

```diff
@@ -46,10 +48,11 @@ const Series = memo(
     const providerValueRef = useRef(null);
 
     const axis = useAxis(axisId);
+    const colorAxis = useColorAxis(colorAxisId);
```

Comment:
Should this be wrapped in a `if (colorAxisId)`?

Reply:
Hooks cannot be run conditionally. They must be run in the same order in every render.
https://reactjs.org/docs/hooks-rules.html